packages/sdk/src/conversation/interface.ts

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Activity, TurnContext } from "botbuilder-core";
+
+/**
+ * The target type where the notification will be sent to.
+ *
+ * @remarks
+ * - "Channel" means to a team channel. (By default, notification to a team will be sent to its "General" channel.)
+ * - "Group" means to a group chat.
+ * - "Person" means to a personal chat.
+ *
+ * @beta
+ */
+export type NotificationTargetType = "Channel" | "Group" | "Person";
+
+/**
+ * Represent a notification target.
+ *
+ * @beta
+ */
+export interface NotificationTarget {
+  /**
+   * The type of target, could be "Channel" or "Group" or "Person".
+   *
+   * @beta
+   */
+  readonly type?: NotificationTargetType;
+
+  /**
+   * Send a plain text message.
+   *
+   * @param text - the plain text message.
+   *
+   * @beta
+   */
+  sendMessage(text: string): Promise<void>;
+
+  /**
+   * Send an adaptive card message.
+   *
+   * @param card - the adaptive card raw JSON.
+   *
+   * @beta
+   */
+  sendAdaptiveCard(card: unknown): Promise<void>;
+}
+
+/**
+ * Interface for a storage provider that stores and retrieves notification target references.
+ *
+ * @beta
+ */
+export interface NotificationTargetStorage {
+  /**
+   * Read one notification target by its key.
+   *
+   * @param key - the key of a notification target.
+   *
+   * @returns - the notification target. Or undefined if not found.
+   *
+   * @beta
+   */
+  read(key: string): Promise<{ [key: string]: any } | undefined>; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
minor: `unknown` is a better choice than `any`.

Reply:
Good catches. Fixed all wordings.